metron-interface/metron-rest/README.md

```diff
@@ -361,6 +363,21 @@ Request and Response objects are JSON formatted.  The JSON schemas are available
     * 200 - Returns sample message
     * 404 - Either Kafka topic is missing or contains no messages
 
+### `POST /api/v1/metaalert/searchByAlert`
+  * Description: Searches meta alerts to find any containing an alert for the provided GUID
+  * Input:
+    * guid - GUID of the alert
+  * Returns:
+    * 200 - Returns the meta alerts associated with this alert
+    * 404 - Either Kafka topic is missing or contains no messages
```

Comment:
Is this a mistake?

Reply:
Absolutely a mistake.  Thought I'd fixed that, but apparently not.